compiler/rustc_borrowck/src/region_infer/graphviz.rs

```diff
@@ -6,7 +6,38 @@ use std::borrow::Cow;
 use std::io::{self, Write};
 
 use super::*;
+use itertools::Itertools;
 use rustc_graphviz as dot;
+use rustc_middle::ty::UniverseIndex;
+
+fn render_outlives_constraint(constraint: &OutlivesConstraint<'_>) -> String {
+    match constraint.locations {
+        Locations::All(_) => "All(...)".to_string(),
```

Comment:
We might as well remove the ellipsis altogether, wdyt?

Reply:
I considered this as my first option, but felt uneasy about leaving something out without showing that’s what I’m doing. Otherwise I don’t think it’s a bad option!